internal/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -230,6 +230,7 @@ func (c *enrollCmd) Execute(ctx context.Context, streams *cli.IOStreams) error {
 	if localFleetServer {
 		// Ensure that the agent does not use a proxy configuration
 		// when connecting to the local fleet server.
+		// TODO At this point remote config has $HOSTNAME:8220 - is that expected or do we part port 8221?
```

Comment:
@michalpristas what was your original throughs when implementing this?

Reply:
in case proxy is configured i want to avoid having internal connection routed through external proxy, that's all